runtime/oti/vmconstantpool.xml

```diff
@@ -50,6 +50,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 	<classref name="java/lang/IllegalThreadStateException"/>
 	<classref name="java/lang/Thread"/>
 	<classref name="java/lang/Thread$FieldHolder" versions="19-"/>
+	<classref name="java/lang/BaseVirtualThread" versions="19-"/>
```

Comment:
To match the RI, `VirtualThread` must be used instead of `BaseVirtualThread`. `BaseVirtualThread` is abstract.

Reply:
I think base virtual thread is correct, all virtual threads subclass it

Reply:
I don't see a specific restriction to use `VirtualThread` in jvmti. So, using `BaseVirtualThread` should be fine, and seems more correct.